### ISSUE TITLE
Getting Started Tutorial Clarity Suggestion

### ISSUE THREAD

**Issue by bessermt** [Microsoft/cpprestsdk#191]
The "Getting Started Tutorial" 

https://github.com/Microsoft/cpprestsdk/wiki/Getting-Started-Tutorial

Says:

To save explicitly typing common namespaces over and over add the following using statements:

using namespace utility;                    // Common utilities like string conversions
using namespace web;                        // Common features like URIs.
using namespace web::http;                  // Common HTTP functionality
using namespace web::http::client;          // HTTP client features
using namespace concurrency::streams;       // Asynchronous streams

I recommend removing this code and using explcit namespaces in the code. Learning a library is significantly more difficult when common standard C++ class names are used (ex. fstream) but actually refer to concurrency::streams::fstream instead of std::fstream. I assume the goal of a Getting Started Tutorial is to make the library's use explicitly clear and not hidden, regardless of the expense of extra typing.

**Comment by ras0219-msft** [comment]
Good point about confusion between `std::fstream` and `concurrency::streams::fstream`. Adding at least one level of qualification to the uses of `fstream` would definitely reduce user confusion. I think it would also be reasonable to qualify `web::uri` and `web::uri_builder` as well.

Some of our types have been explicitly named with `using namespace` in mind, such as those in `web::http::client` (`http_client` and `http_client_config`), so that particular `using namespace` makes the example more concise without sacrificing clarity.
